src/appleseed-max-impl/appleseedrenderer/renderersettings.cpp

```diff
@@ -170,9 +176,23 @@ void RendererSettings::apply_settings_to_final_config(asr::Project& project) con
     params.insert_path("passes", m_passes);
     params.insert_path("shading_result_framebuffer", m_passes == 1 ? "ephemeral" : "permanent");
 
-    params.insert_path("uniform_pixel_renderer.samples", m_pixel_samples);
-    if (m_pixel_samples == 1)
-        params.insert_path("uniform_pixel_renderer.force_antialiasing", true);
+    if (m_sampler_type == 0)
+    {
+        params.insert_path("pixel_renderer", "uniform");
+        params.insert_path("uniform_pixel_renderer.samples", m_uniform_pixel_samples);
+        if (m_uniform_pixel_samples == 1)
+            params.insert_path("uniform_pixel_renderer.force_antialiasing", true);
+    }
+    else
+    {
+        params.insert_path("pixel_renderer", "");
```

Comment:
You don't need to set a `pixel_renderer` param in this case.

Reply:
I exported a test scene from as_studio and it had this line included.
I was not sure if it needed to be added in the project file. I have deleted it now.